<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Редактировани</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionFont"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/icons/images/new.png</normaloff>:/icons/images/new.png</iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="toolTip">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/icons/images/open.png</normaloff>:/icons/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/icons/images/save.png</normaloff>:/icons/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="toolTip">
    <string>Закрыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/icons/images/copy.png</normaloff>:/icons/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/icons/images/paste.png</normaloff>:/icons/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="toolTip">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/icons/images/cut.png</normaloff>:/icons/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="toolTip">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="toolTip">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="toolTip">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О приложении</string>
   </property>
   <property name="toolTip">
    <string>О приложении</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>Информация Qt</string>
   </property>
   <property name="toolTip">
    <string>Информация Qt</string>
   </property>
  </action>
  <action name="actionSelect_font">
   <property name="text">
    <string>Выбрать шрифт</string>
   </property>
   <property name="toolTip">
    <string>Выбрать шрифт</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Шрифт</string>
   </property>
   <property name="toolTip">
    <string>Шрифт</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="toolTip">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="texteditor.qrc"/>
 </resources>
 <connections/>
</ui>
